scripts/inline_comments_data_collection.py

```diff
@@ -136,18 +107,25 @@ def process_comments(limit, diff_length_limit):
             if not most_recent_update:
                 continue
 
-            fix_patch_id, revision_phid = get_diff_info_from_phid(
-                most_recent_update["fields"]["new"]
-            )
+            fix_patch_id = diff_phid_to_revisions_map.get(
+                most_recent_update["fields"].get("new"), [None]
+            )[0]
 
-            # If the most recent patch is the original patch itself, skip it
-            if fix_patch_id == patch_id:
+            # If the  most recent patch doesn't exist or is the original patch itself, skip it
+            if not fix_patch_id or fix_patch_id == patch_id:
                 continue
 
-            revision_id, bug_id = find_details_from_revision_phid(phid=revision_phid)
+            revision_info = diff_phid_to_revisions_map.get(
+                most_recent_update["fields"].get("new"), [None, {}]
+            )[1]
+            revision_phid = revision_info.get("phid")
+            revision_id = revision_info.get("id")
```

Comment:
`revision_info` could just be `diff_id_to_revisions_map[patch_id]`, if you add it at line 97 then you could do:
```Python
revision_info = diff_id_to_revisions_map[patch_id]
transactions = revision_info["transactions"]
```

Reply:
And if you do this, you could change `diff_phid_to_revisions_map` to only store the ID, renaming it to `diff_phid_to_id`:
```Python
diff_phid_to_revision[transaction["fields"]["diff"]["phid"]] = transaction["fields"]["diff"]["id"]
```

Reply:
Changed here: 255c1d6e24c63fa71cde29861300551db5b89f70. Thanks for pointing that out!